LibreHardwareMonitorLib/Hardware/Controller/Corsair/CommanderProDevice.cs

```diff
@@ -0,0 +1,94 @@
+﻿using HidSharp;
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace LibreHardwareMonitor.Hardware.Controller.Corsair
+{
+    internal delegate void CommanderProAction(HidStream stream, byte[] readBuffer, byte[] writeBuffer);
+
+    internal class CommanderProDevice : IDisposable
+    {
+        private readonly TimeSpan _defaultTimeout = TimeSpan.FromSeconds(3);
+        private readonly HidStream _hidStream;
+        private byte[] _writeBuffer = new byte[63];
```

Comment:
I think this was a typo in one of the protocol descriptions and must be 64 bytes long